### DISCUSSION TITLE
Fuel burn

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#4442] [General]
Hi developers,

I know that you changed the engine model. Is that part finished or still in progress? I am looking especially to the fuel burn inflight and figured out, that this is right now to low for the A 320 neo. The average consumption in round about 2.100 kg/hr (A 320 "classic" round about 2.500 kg/hr). 

If I calculate the average from legs with head and tailwind I have a consumption of 1.900 kg/hr ... Still missing ca. 200 kg/hr.

Greets klaus

**Comment by hotshotp** [comment]
it is early in development right now so some parts of it are not too accurate

**Comment by CptBayer** [comment]
just for information... Look at the wind and the fuelburn. Very low values. With 70 kts from the same direction I monitored more fuel burn. Round about 20 kg/h per engine .... In general to low burn. Looks like, that the wind is not yet implemented in the whole consumption topic.... as you wrote, early stage :-) .... 

![grafik](https://user-images.githubusercontent.com/76531893/115362744-23b93600-a1c2-11eb-9c70-c031afd1f31c.png)

greets Klaus

**Reply by CptBayer** [reply]
Hi developers,

I regognized a change in the eengine model .... right now it seems a bit to high .. :-p .. 17 kts headwind. Not a lot ... But I think developing is still in progress. 

![grafik](https://user-images.githubusercontent.com/76531893/116061301-fcabaa00-a682-11eb-9cea-e5584a9da1b9.png)

Greets Klaus

**Reply by CptBayer** [reply]
at FL 160 with 3 kts head wind and a GW of 61 t ... I had to do a refueling in the air due to fuel BIAS from the last engine model :-P ... 

![grafik](https://user-images.githubusercontent.com/76531893/116063676-8a889480-a685-11eb-9f5e-2e9fe0e671fd.png)
